app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
```

Comment:
AR query looks good! I still don't understand why it's merchant_id but if it's working that's great!